nextstrain_profiles/nextstrain-scicore/cluster.json

```diff
@@ -20,12 +20,16 @@
     "time": "05:59:00"
   },
   "align": {
-    "n": 16,
-    "mem": "128G",
+    "n": 8,
+    "mem": "16G",
     "qos": "6hours",
     "time": "05:59:00"
   },
   "proximity_score": {
     "mem": "64G"
```

Comment:
Can this be reduced to `2G` now?

Reply:
I'd make if 4 or 8, but certainly doesn't need to be 64 anymore. 
```
(nextstrain) [neher@login20 ncov2]$ more benchmarks/proximity_score_*
::::::::::::::
benchmarks/proximity_score_africa_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
268.1170        0:04:28 1178.01 1767.21 1166.00 1167.89 0.00    0.00    95.12   255.49
::::::::::::::
benchmarks/proximity_score_asia_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
315.9518        0:05:15 1161.40 1773.18 1154.78 1157.08 0.00    0.00    90.20   285.40
::::::::::::::
benchmarks/proximity_score_europe_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
281.7032        0:04:41 1211.82 1846.09 1199.81 1201.70 0.00    0.00    90.53   255.44
::::::::::::::
benchmarks/proximity_score_north-america_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
333.0056        0:05:33 1176.44 1763.07 1174.81 1178.41 0.00    0.00    94.62   315.55
::::::::::::::
benchmarks/proximity_score_oceania_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
230.3788        0:03:50 726.38  1234.78 714.36  716.25  0.00    0.00    97.65   225.43
::::::::::::::
benchmarks/proximity_score_south-america_region_late.txt
::::::::::::::
s       h:m:s   max_rss max_vms max_uss max_pss io_in   io_out  mean_load       cpu_time
249.8558        0:04:09 701.34  1252.14 689.50  693.10  0.00    0.00    89.35   223.41
```

Reply:
I left this at 4G, but I added a note that the memory use seems to scale at about 0.15 MB * chunk size (based on only on two data points of chunk size = 5000 and 10000). Eventually we could choose to implement the mem_mb as a function of chunk size to make this more flexible.